docs/admin/admission-controllers.md

```diff
@@ -16,16 +16,27 @@ title: Using Admission Controllers
 
 An admission control plug-in is a piece of code that intercepts requests to the Kubernetes
 API server prior to persistence of the object, but after the request is authenticated
-and authorized.  The plug-in code is in the API server process
-and must be compiled into the binary in order to be used at this time.
+and authorized.  The plug-ins discussed in this document are compiled into the
+API server process and may only be configured by the cluster administrator. As of 1.9, [user provided
+webhooks](/docs/admin/extensible-admission-controllers.md#external-admission-webhooks)
+are in beta; these are dynamically configurable.
```

Comment:
_runtime configurable_ is more specific than _dynamically configurable_?

Reply:
Confusing to talk about plugins and then suddenly about webhooks. Question a read might have: is a webhook a plugin?